science/ngspice/Portfile

```diff
@@ -127,25 +124,34 @@ subport ngspice-docs {
 
 subport ngspice-lib {
     description         ngspice dynamic library
-    long_description    ${description}
+    long_description    {*}${description}
 
-    # freetype2 headers are not found by default
-    # we could fix the header path in configure.ac, but then need autorecconf, etc
-    configure.cppflags-prepend -I${prefix}/include/freetype2
+	# freetype2 headers are not found by default
+	configure.cppflags-prepend -I${prefix}/include/freetype2
 
-    # TASK_BASIC_INFO_COUNT and friends were renamed in 10.8
-    patchfiles-append   patch-ngspice-older-MACH-defines.diff
+    depends_lib-append  port:autoconf \
+    					port:automake \
+    					port:bison \
```

Comment:
Same remarks as above: use spaces instead of tabs, and distinguish between library and build dependencies. Also, avoid duplication. If the same dependencies are needed for multiple subports within a port, write them only once in a common section of the portfile.

Reply:
Won't be a problem in the future.  When I reboot into English, things behave better.